charts/consul/values.yaml

```diff
@@ -2195,6 +2195,17 @@ connectInject:
         # @type: string
         service: null
 
+      # The resource settings for Pods handling traffic for Gateway API.
+      # @recurse: false
+      # @type: map
+      resources:
+        requests:
+          memory: "100Mi"
+          cpu: "100m"
+        limits:
+          memory: "100Mi"
+          cpu: "100m"
```

Comment:
I see these were just moved over from before, but are these the default resource values we want to go out the door with? Probably worth running past @Jeff-Apple 

Edit: If we were to change it here, then we probably also want to change it in `defaultResourceRequirements` below

Reply:
That's a great call-out. I'm not sure what the best limit would be for these.

Reply:
These values are the same as the defaults for Ingress Gateway -- so I think that's a good sign that they are solid defaults here.

Reply:
I checked this out with @lkysow and he gave the thumbs up on these values.

Reply:
Thanks @t-eckert !